stub-connector/src/test/java/uk/gov/ida/notification/apprule/base/StubConnectorAppRuleTestBase.java

```diff
@@ -21,18 +23,19 @@
 
 public class StubConnectorAppRuleTestBase extends AbstractSamlAppRuleTestBase {
 
-    private static final KeyStoreResource METADATA_TRUSTSTORE = createMetadataTruststore();
+    protected static final String ENTITY_ORG_NAME = "stub country org name";
+    protected static final String ENTITY_ORG_DISPLAY_NAME = "stub country org display name";
+    protected static final String ENTITY_ORG_URL = "http://stub-connector/homepage";
     protected static final String METADATA_CERTS_PUBLISH_PATH = "/proxy-node-md-certs-publish-path";
     protected static final String METADATA_PUBLISH_PATH = "/stub-connector-md-publish-path";
     protected static final String ENTITY_ID = "http://stub-connector/Connector";
     protected static final String ACS_URL = "http://stub-connector/SAML2/Response/POST";
-    public static final String ENTITY_ORG_NAME = "stub country org name";
-    public static final String ENTITY_ORG_DISPLAY_NAME = "stub country org display name";
-    public static final String ENTITY_ORG_URL = "http://stub-connector/homepage";
+
+    private static final KeyStoreResource METADATA_TRUSTSTORE = createMetadataTruststore();
 
     private Map<String, NewCookie> cookies;
 
-    protected String getEidasRequest(StubConnectorAppRule stubConnectorAppRule) throws URISyntaxException {
+    protected String getEidasRequest(AppRule<StubConnectorConfiguration> stubConnectorAppRule) throws URISyntaxException {
```

Comment:
Again here, the only two children of `StubConnectorAppRuleTestBase` both have exactly the same setup for AppRules and ClientRules. They could be pulled back to the base class which would simplify all these methods.

Reply:
Nope - that will break the tests due to how Dropwzard AppRules work in tests